### PR TITLE
More layout touchups for the campaign template.

### PR DESCRIPTION
**Pull request by DFurnes**
### What's this PR do?

This pull request contains more touch-ups to the campaign template layout, notably updating the CampaignPage and ContentBlock to correctly align to the grid (using Tailwind's new [grid helpers](https://tailwindcss.com/docs/grid-template-columns/)!)

### How should this be reviewed?

Here are some screenshots, but also feel free to bop around on the review app!

Screen Shot 2020-02-07 at 3 18 14 PM

Screen Shot 2020-02-07 at 3 20 54 PM

Screen Shot 2020-02-07 at 3 21 51 PM

### Any background context you want to provide?

This fixes up a little layout bug introduced in #1910. I'll fix up the Social Drive block layout (where it may expand too far now if it doesn't have a click counter) in a follow-up PR.

### Relevant tickets

References [Pivotal #171161967](https://www.pivotaltracker.com/story/show/171161967).

### Checklist

- [x] This PR has been added to the relevant Pivotal card.
- [ ] Documentation added for new features/changed endpoints.
- [ ] Added screenshots of front-end changes on small, medium, and large screens.
- [ ] Added appropriate feature/unit tests.

### CODE REVIEW

File: resources/assets/components/blocks/ContentBlock/ContentBlock.js [hunk 1]
```diff
@@ -39,23 +39,28 @@ const ContentBlock = props => {
   const contentNode = content ? <TextContent>{content}</TextContent> : null;
 
   return (
-    <div className={classnames('content-block', className)}>
```
Comment (mendelB) [inline resources/assets/components/blocks/ContentBlock/ContentBlock.js]: Can we delete the `content-block.scss` file now? 🗑

File: resources/assets/components/blocks/ContentBlock/ContentBlock.js [hunk 2]
```diff
@@ -39,23 +39,28 @@ const ContentBlock = props => {
...
+          >
+            <LazyImage
+              src={contentfulImageUrl(image.url, '600', '600', 'fill')}
+              alt={image.description || 'content-block'}
+            />
+          </div>
+        ) : null}
+
+        <div className="col-span-2 order-1">{contentNode}</div>
+      </div>
```
Comment (mendelB) [inline resources/assets/components/blocks/ContentBlock/ContentBlock.js]: This is hot

File: resources/assets/components/pages/CampaignPage/CampaignPageContent.js [hunk 3]
```diff
@@ -44,21 +44,21 @@ const CampaignPageContent = props => {
...
-                ImagesBlock: 'w-full',
-                PostGalleryBlock: 'w-full',
-                PhotoSubmissionBlock: 'w-full',
-                SocialDriveBlock: 'w-full',
-              }}
-            />
-          </div>
+          <ContentfulEntryLoader
+            key={block.id}
+            id={block.id}
```
Comment (mendelB) [inline resources/assets/components/pages/CampaignPage/CampaignPageContent.js]: Just a heads up that those 👻tests which scope for the block ID will need to be updated!

File: resources/assets/components/blocks/ContentBlock/ContentBlock.js (hunk 1 above)
Comment (DFurnes) [reply resources/assets/components/blocks/ContentBlock/ContentBlock.js]: OH WE SURE AS HECK CAN! 🔥

File: resources/assets/components/pages/CampaignPage/CampaignPageContent.js (hunk 3 above)
Comment (DFurnes) [reply resources/assets/components/pages/CampaignPage/CampaignPageContent.js]: Yeah defs. I'll take care of updating those on QA (and prod) once this is merged! I added a `data-contentful-id` attribute in 27eb42c that we can use in lieu of the IDs.

Comment (mendelB) [reply resources/assets/components/pages/CampaignPage/CampaignPageContent.js]: Ooo nice!!

File: resources/assets/components/blocks/ContentBlock/ContentBlock.js (hunk 1 above)
Comment (DFurnes) [reply resources/assets/components/blocks/ContentBlock/ContentBlock.js]: Removed this in 37193fd0!

**Pull request merged**